packages/components/src/components/Heading/index.ts

```diff
@@ -7,14 +7,52 @@ export const HeadingComponentDefinition: ComponentDefinition = {
   id: 'heading',
   name: 'Heading',
   category: 'Contentful',
-  builtInStyles: ['cfMargin', 'cfPadding'],
+  builtInStyles: [
+    'cfMargin',
+    'cfPadding',
+    'cfFontWeight',
```

Comment:
Are font size, line height, and bold not valid options because they get overridden no matter what you picked due to the default values in the variables section?

Reply:
If those 3 options are added to `builtInStyles` array, the objects in `variables` will be ignored (so it will get the standard default values instead of custom ones.